tensorboard/defs/defs.bzl

```diff
@@ -72,8 +71,8 @@ def tf_js_binary(
         # the global level and tends to overwrite `window` functions. "iife" is
         # just a thin wrapper around "esm" (it adds 11 bytes) and doesn't
         # suffer from the same overwriting problem.
-        format="iife",
-        minify= False if dev_mode_only else True,
+        format = "iife",
```

Comment:
You have unrelated changes here.

Reply:
The observer effect (aka ide autoformat) is alive and strong. Sorry I missed this.